kuma/static/styles/components/payments/form.scss

```diff
@@ -16,13 +16,14 @@
     }
 
     .legal-copy {
+        display: inline-block;
         font-size: 12px;
         font-weight: normal;
         letter-spacing: .1px;
         line-height: 1.35;
         margin-bottom: 18px;
 
-        @media #{$mq-small-desktop-and-up} {
+        @media #{$mq-large-desktop-and-up} {
```

Comment:
I don't get this change. Is the text going to be centered on small desktop or something?

Reply:
The alignment existed before. I simply needed to have the media query kick in a bit later due to the addition of the `max-width`